grpc-server/template_test.go

```diff
@@ -84,22 +158,24 @@ func TestCreateTemplate(t *testing.T) {
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
 			s := testServer(tc.args.db)
```

Comment:
Let's do a test, do you mind to add at line 160:

`t.Parallel()`

Thanks

Reply:
I don't think so, because clearTemplateDB is being called in each test run, which is a global var.

Reply:
That's why I left a comment. We should not do that

Reply:
Ah, yes I agree. {Less,No} global vars :allthethings:!

Reply:
Done